subsystem_controllers/config/guidance_controller_config.yaml

```diff
@@ -23,22 +23,24 @@
       - /guidance/plugins/inlanecruising_plugin
       - /guidance/pure_pursuit_wrapper
       - /guidance/yield_plugin
+      - /guidance/plugins/platoon_control
```

Comment:
I wouldn't probably add platooning to required plugins

Reply:
Related issue here: https://github.com/usdot-fhwa-stol/carma-platform/issues/2383